test/functional/hostgroups_controller_test.rb

```diff
@@ -42,13 +42,13 @@ def test_edit
 
   def test_update_invalid
     Hostgroup.any_instance.stubs(:valid?).returns(false)
-    put :update, {:id => Hostgroup.first, :hostgroup => {}}, set_session_user
+    put :update, {:id => Hostgroup.first, :hostgroup => { :name => Hostgroup.first.name }}, set_session_user
     assert_template 'edit'
   end
 
   def test_update_valid
     Hostgroup.any_instance.stubs(:valid?).returns(true)
-    put :update, {:id => Hostgroup.first, :hostgroup => {}}, set_session_user
+    put :update, {:id => Hostgroup.first, :hostgroup => { :name => Hostgroup.first.name }}, set_session_user
```

Comment:
This seems a strange thing to do - is it passing validation only because of the stub?  Surely it'd have a name conflict.